src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -427,6 +460,7 @@ static void Main()
             Assert.Null(model.GetTypeInfo(def).Type);
             Assert.Null(model.GetSymbolInfo(def).Symbol);
             Assert.Null(model.GetDeclaredSymbol(def));
+            Assert.Equal(0, model.GetConstantValue(def).Value);
```

Comment:
>Assert.Equal(0, model.GetConstantValue(def).Value); [](start = 12, length = 51)

What type is returned by SemanticModel for the "def" node here? That type and the type of the constant value should agree.